resources/lib/utils.py

```diff
@@ -100,26 +100,6 @@ def reformat_image_url(url):
     return ''
 
 
-def program_to_url(program, url_type):
```

Comment:
This was entirely unused, so I removed it rather than support vrtmax urls as well.